packages/transfer/src/translators/reactjs/state/state.ts

```diff
@@ -33,6 +33,8 @@ const transferReducer = (state: TransferState, action: TransferAction) => {
         draft.selectedTokenId = defaultTransferState.selectedTokenId
         draft.memo = defaultTransferState.memo
         draft.unsignedTx = castDraft(defaultTransferState.unsignedTx)
+        draft.selectedTargetIndex = defaultTransferState.selectedTargetIndex
```

Comment:
bug when resetting state, was not setting back some properties cc: @banklesss  (maybe there is a better way of settings the entire draft as initialState)

Reply:
need to have a look

Reply:
amazing catch